docs/what-is-rocm.rst

```diff
@@ -121,7 +121,7 @@ Compilers
 
   "`AOMP <https://github.com/ROCm/aomp/>`_", "Scripted build of `LLVM <https://github.com/ROCm/llvm-project>`_ and supporting software"
   "`FLANG <https://github.com/ROCm/flang/>`_", "An out-of-tree Fortran compiler targeting LLVM"
-  "`hipCC <https://github.com/ROCm/HIPCC>`_ ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
+  "`hipCC <https://rocm.docs.amd.com/projects/HIPCC/en/latest/index.html>`_ ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
```

Comment:
Can you implement using the doc role, like the examples above in the ML&CV or Primitives section?  

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role

Reply:
I usually refer to https://github.com/ROCm/rocm-docs-core/blob/develop/src/rocm_docs/data/projects.yaml

Find the HIPCC key and use the same capitalization/spelling in the `:doc:` directive

Reply:
```suggestion
":doc:`hipCC <hipcc:index>`", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
```

Reply:
@randyh62 for me the suggested version worked.

Reply:
@peterjunpark was faster.